src/content/docs/alerts-applied-intelligence/applied-intelligence/postmortems-applied-intelligence.mdx

```diff
@@ -22,7 +22,7 @@ import accountsPostmortemEdit from 'images/accounts_screenshot-full_postmortem-e
 This feature is still in development, but we encourage you to try it out.
 </Callout>
 
-With applied intelligence's postmortems feature, events, metrics, and metadata are automatically collected to create a template you can use to run an incident postmortem for your team.
+With alert's postmortems feature, events, metrics, and metadata are automatically collected to create a template you can use to run an incident postmortem for your team.
```

Comment:
I wonder if this should be

Alerts' 

Is the product name "Alerts" in the plural?

Reply:
It is alerts in the plural!